plugin/user_plugin_server.py

```
@@ -0,0 +1,558 @@
+            desc = pdata.get("description", "")
+            version = pdata.get("version", "0.1.0")
+            entry = pdata.get("entry")
+            if not entry or ":" not in entry:
+                logger.warning("plugin.toml %s has invalid entry=%r, skipping", toml_path, entry)
+                continue
+
+            module_path, class_name = entry.split(":", 1)
+            mod = importlib.import_module(module_path)
+            cls = getattr(mod, class_name)
+
+            # 实例化插件；如果将来想传 ctx，可以改成 cls(ctx)
```

Ruff (0.14.7)

Comment contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF003)